src/NuGet.Core/NuGet.Protocol/Strings.resx

```diff
@@ -467,7 +471,7 @@ The "ms" should be localized to the abbreviation for milliseconds.</comment>
 {1} is the exception ToString() result</comment>
   </data>
   <data name="Log_LocalSourceNotExist" xml:space="preserve">
-    <value>The local source '{0}' doesn't exist.</value>
+    <value>The local source or fallback folder '{0}' doesn't exist. Create directory, or fix packageSource/fallbackPackageFolder declarations.</value>
```

Comment:
Probably should match the exact name in the NuGet.Config schema. So i'd suggest using plural here.

Reply:
you are suggesting to change the end of that from:
...or fix packageSource/fallbackPackageFolder declarations
to:
..or fix packageSources/fallbackPackageFolders declarations

i'm not sure if i agree. we want them to fix the declaration of the package source, not the package sources.
it is a child element inside of those plural words, yes.

the NU1302 doc page can explain it very well...giving example, but i think i like singular better.

Reply:
please reactivate if you want to discuss given my explanation. happy to.

Reply:
I think `package source` is correct, or `packageSources` is correct. Camelcase, package source is not used in a configuration, so I don't see why we can't just use normal wording. 

Reactivating cause I'm curious what others think.
Honestly reading it again, I think just saying `package source` is better. 

cc @JonDouglas @chgill-MSFT hoping for your help here.

Reply:
`Package Source(s)` perhaps?

Reply:
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/s/s-es suggests `one or more package sources` instead of using `(s)`

Reply:
how do people like this?

"The local source or fallback folder '{0}' doesn't exist. Create directory, or fix the package source or fallback folder declaration."

Reply:
`The local source or fallback folder '{0}' doesn't exist. Create the directory if it doesn't exist, ensure the package source is reachable, or define a fallback folder.`

Reply:
RE: our conversation **"The "ResolvePackageAssets" task failed unexpectedly (FallbackPackageFolders / Xamarin.Offline.config)"**
Can we use this message to be more clear in VS when the fallback folder doesn't exist? (https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1258922)